src/azure-cli/azure/cli/command_modules/security/custom.py

```diff
@@ -305,3 +306,21 @@ def create_security_workspace_setting(client, resource_name, target_workspace):
 def delete_security_workspace_setting(client, resource_name):
 
     client.delete(resource_name)
+
+
+# --------------------------------------------------------------------------------------------
+# Security ATP
+# --------------------------------------------------------------------------------------------
+
+
+def get_atp_setting(client, resource_group_name, storage_account_name):
+
+    return client.get('subscriptions/{}/resourceGroups/{}/providers/Microsoft.Storage/storageAccounts/{}'
```

Comment:
Consider extracting construction of the resource ID to a utility function, since it's being reused. But not that important.

Reply:
Done :)